builtin/game/falling.lua

```diff
@@ -27,12 +27,21 @@ core.register_entity(":__builtin:falling_node", {
 			is_visible = true,
 			textures = {node.name},
 		})
+		local def = core.registered_nodes[node.name]
+		if node.param2 ~= 0 then
```

Comment:
`if def and node.param2 ~= 0`
For the special case when falling node entities are saved, the mod gets disabled and this node becomes an unknown one.

Reply:
And this is is a problem … how?
Is the node table in a weird format when this happens?

Reply:
Yes, `def` is nil if it's an known node.

Reply:
^ unknown?